app/views/hosts/_operating_system.html.erb

```diff
@@ -9,26 +9,18 @@
 
 <div id='provisioning_method' <%= display? (@host.capabilities.size < 2)  %> >
   <%= field(f, :provision_method, :label => _('Provisioning Method'), :required => true) do
```

Comment:
add `:size => 'col-md-10'` in here to make the field full width, this will stop ugly wrapping of the radio buttons when adding new methods:

![screenshot from 2015-12-07 14-25-26](https://cloud.githubusercontent.com/assets/482089/11629157/69c85574-9cee-11e5-89a5-746640ac4442.png)